if.go

```diff
@@ -77,3 +78,8 @@ func (ifce *Interface) IsTAP() bool {
 func (ifce *Interface) Name() string {
 	return ifce.name
 }
+
+// FD returns the underlying File object, which will be needed to ioctl later
+func (ifce *Interface) FD() *os.File {
```

Comment:
I don't see this used anywhere inside the package. I assume you need it in some package that imports this package. Would you mind sharing your use case? I was under the impression that the `ReadWriteCloser` interface would be sufficient.

Reply:
Indeed, I am using it to later run `ioctl`s on the interface, as I was not entirely sure on the solidit of turning the `ReadWriteCloser` back into an `*os.File`. In fact, for my particular use case, that was not possible at all due to the Interface object being hidden behind a Go interface.

Reply:
It'd be nice to not export the `*os.File` type. I think any syscalls should be handled within this package. An important reason that this package exists is to hide away all the lower level calls so a user of this package don't need to touch e.g. file descriptors.